server/etcdserver/v3_server_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The etcd Authors
```

Comment:
@ahrtr great work. One comment here - can we move the test func in `server/etcdserver/server_test.go` instead of creating this new test file? (rename the func to reflect v3 similar to other funcs in the server_test.go) Thanks!

Reply:
Thanks & resolved.